src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/locationsinsideprison/LocationsInsidePrisonClient.kt

```diff
@@ -19,3 +22,41 @@ class LocationsInsidePrisonClient(private val locationsInsidePrisonApiWebClient:
     .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
     .block()
 }
+
+@Component
+class LocationValidator(private val locationsInsidePrisonClient: LocationsInsidePrisonClient) {
```

Comment:
So not 100% convinced on this but I thought I'd try a slim down the validation login the in the creation service.  It is likely similar validation will be required for edit/update.  Keeps the testing simpler in there services as well.